docs/api/covidcast-signals/safegraph.md

```diff
@@ -4,18 +4,21 @@ parent: Data Sources and Signals
 grand_parent: COVIDcast API
 ---
 
-# SafeGraph Mobility
+This data source uses data reported by [SafeGraph](https://www.safegraph.com/)
+using anonymized location data from mobile phones. We provide multiple signals based on different datasets provided to eligible researchers who obtain an API key by SafeGraph.
+
+
+# SafeGraph Social Distancing Metrics
 
 * **Source name:** `safegraph`
 * **First issued:** 23 June 2020
 * **Number of data revisions since 23 June 2020:** 0
 * **Date of last change:** never
 * **Available for:** county, state (see [geography coding docs](../covidcast_geography.md))
 
-This data source uses data reported by [SafeGraph](https://www.safegraph.com/)
-using anonymized location data from mobile phones. SafeGraph provides [social
-distancing metrics](https://docs.safegraph.com/docs/social-distancing-metrics)
-to eligible researchers who obtain an API key. SafeGraph provides this data for
+Data source based on [social
+distancing metrics](https://docs.safegraph.com/docs/social-distancing-metrics). 
+SafeGraph provides this data for
 individual census block groups, using differential privacy to protect the
```

Comment:
"using differential privacy to protect individual people's privacy in data"

Make it consistent with the other portion. Also, can you clarify what you mean by "in data"? 
1. does it just mean data privacy? If so it can be reworded as 
    "using differential privacy to protect individual people's data privacy"

2.  if it means something else then maybe we should make sure we clarify what sort of data

Reply:
Thanks. Will reword it.